src/main/java/com/epam/aidial/core/service/InvitationService.java

```diff
@@ -170,22 +170,35 @@ public void cleanUpPermissions(
         });
     }
 
-    public void moveResource(String bucket, String location, ResourceDescription source, ResourceDescription destination) {
-        ResourceDescription resource = ResourceDescription.fromDecoded(ResourceType.INVITATION, bucket, location, INVITATION_RESOURCE_FILENAME);
-        resourceService.computeResource(resource, state -> {
+    public void moveResource(ResourceDescription source, ResourceDescription destination) {
+        if (!source.isPrivate()) {
```

Comment:
Should we add a warning message?

Reply:
Why? Now it is a normal path when we move resources from public bucket.

Reply:
Reworked to make it more clear.